presto-native-execution/presto_cpp/main/http/HttpClient.cpp

```diff
@@ -149,6 +155,16 @@ class ResponseHandler : public proxygen::HTTPTransactionHandler {
 
   void onBody(std::unique_ptr<folly::IOBuf> chain) noexcept override {
     if ((chain != nullptr) && !response_->hasError()) {
+      if (type_ == common::PRESTO_ANNOUNCER) {
```

Comment:
We can't get this stats from proxygen stats filer? @amitkdutta Doesn't the response distribution correspond to the ones collected at the server side? Thanks!

Reply:
Probably not onbody. We can get num requests

Reply:
We need to understand this PR. Here is what we export today:
- Requests
- Requests in flight (active requests)
- counter for each http code
- Request bytes
- Response bytes
- err_none
- err_read
- err_write
- err_timeout
- err_streamabort
- err_connectionreset
- and a few other erros
These are basic http counters used in almost every server. Do we really need something more?

Reply:
Discussed with Amit yesterday, we only have http server stats, not http client stats